pfcli/service/client/project.py

```diff
@@ -237,12 +237,12 @@ def __init__(self, template: Template, **kwargs):
         self.initialize_project()
         super().__init__(template, project_id=self.project_id, **kwargs)
 
-    def list_vm_locks(self, vm_config_id: T, lock_type: LockType) -> List[dict]:
+    def list_vm_locks(self, vm_config_id: T, lock_status: LockStatus) -> List[dict]:
         response = safe_request(self.list, err_prefix="Failed to inspect locked VMs.")(
-            path=f"{vm_config_id}/vm_lock/", params={"lock_type": lock_type}
+            path=f"{vm_config_id}/vm_lock/", params={"status": lock_status}
         )
         return response.json()
 
     def get_active_vm_count(self, vm_config_id: T) -> int:
-        vm_locks = self.list_vm_locks(vm_config_id, LockType.ACTIVE)
+        vm_locks = self.list_vm_locks(vm_config_id, LockStatus.ACTIVE)
```

Comment:
When there are 16 VM quotas and
12 VMs are running, and 4 VMs are being deleted.

If a user runs `pf vm list`, He would see that 4 VMs are available, and run a new job.
However, PFT server may reject the job because no available VM exists.

To prevent such cases, should we inform users that some VM's are in deletion mode? (that VMs will be available in a few minutes)